src/svgui/svgui.ts

```diff
@@ -706,6 +768,7 @@ export class NamedBox extends UIElement {
             captionText: '',
             borderThickness: 1,
             child: null,
+            image: null,
```

Comment:
I don't think that it's a good idea to modify `NamedBox` with additional features like image.
The right way to do this is to compose image inside `NamedBox` as child of layout component like `UIList`.